### PR TITLE
Ported NowJson library to Lizard

### PR DESCRIPTION
**Pull request by pacmancoder**
PR contains integrated NowJson library; Some changes were made to remove WaykNow dependencies:
- Removed NowMath.h and NowFile.h dependencies (From NowMath.h only transitive stdint.h include was used, NowFile.h routines were replaced with Lizard file functions)
- strtod_rs was replaced with normal strtod - however, this a little bit concerns me, is strtod_rs was used just for the better performance? It looks like after replacing strtod_rs with strtod everything still looks fine.

### CODE REVIEW

File: src/LzJson.c [hunk 1]
```diff
@@ -0,0 +1,1798 @@
...
+    if (length > 2 && !strncmp(string, "-0", 2) && string[2] != '.')
+        return 0;
+    while (length--)
+        if (strchr("xX", string[length]))
+            return 0;
+    return 1;
+}
+
+static char * read_file(const char * filename) {
+    FILE *fp = LzFile_Open(filename, "rb");
```
Comment (pacmancoder) [inline src/LzJson.c:228]: Replaced all NowFile calls with LzFile

File: src/LzJson.c [hunk 2]
```diff
@@ -0,0 +1,1798 @@
...
+    } else if (strncmp("false", *string, false_token_size) == 0) {
+        *string += false_token_size;
+        return json_value_init_boolean(0);
+    }
+    return NULL;
+}
+
+static JSON_Value * parse_number_value(const char **string) {
+    char* end_ptr = NULL;
+    double number = strtod(*string, &end_ptr);
```
Comment (pacmancoder) [inline src/LzJson.c]: Replaced strtod_rs with strtod. strtod_rs was returning parsed chunk size via second argument directly, while strtod sets the second argument as a pointer to the symbol after parsed data, so length of parsed chunk still can becalculated.

**Comment by awakecoding** [comment]
@pacmancoder strtod was replaced because the regular one is locale-dependent. However, I don't think this should cause issues for the type of usage we're going to need it for, IIRC it is mostly for floating points.

**Comment by awakecoding** [comment]
can you prefix all json_ symbols (all those that aren't 'static') with lz_? For instance, json_object_get_value would become lz_json_object_get_value, etc. We do link to lizard from Wayk Now, so any non-static symbol built in both libraries would likely cause duplicated symbols. Type definitions shouldn't really cause issues, only the non-static function names.

**Comment by pacmancoder** [comment]
@awakecoding Yes, I already working on this. 👍

**Comment by pacmancoder** [comment]
@awakecoding non-static symbols were renamed, also the version was bumped up. PR is ready for review & merge

**Pull request merged**